### PR TITLE
Add "I can eat glass" to corpus

### PR DESCRIPTION
**Pull request by jystic**
I saw @thumphries using these translations in hydrant, thought they would be nice as a corpus.

! @thumphries @charleso @erikd-ambiata @amosr

### CODE REVIEW

**Comment by charleso** [comment]
👍

File: disorder-corpus/src/Disorder/Corpus.hs [hunk 1]
```diff
@@ -357,3 +361,180 @@ boats = [
...
+  , "Mi posso magnare el vetro, no'l me fa mae." -- Venetian
+  , "Pòsso mangiâ o veddro e o no me fà mâ." -- Zeneise (Genovese)
+  , "Puotsu mangiari u vitru, nun mi fa mali. " -- Sicilian
+  , "Jau sai mangiar vaider, senza che quai fa donn a mai. " -- Romansch (Grischun)
+  , "Pot să mănânc sticlă și ea nu mă rănește." -- Romanian
+  , "Mi povas manĝi vitron, ĝi ne damaĝas min. " -- Esperanto
+  , "Mý a yl dybry gwéder hag éf ny wra ow ankenya." -- Cornish
+  , "Dw i'n gallu bwyta gwydr, 'dyw e ddim yn gwneud dolur i mi." -- Welsh
+  , "Foddym gee glonney agh cha jean eh gortaghey mee." -- Manx Gaelic
+  , "᚛᚛ᚉᚑᚅᚔᚉᚉᚔᚋ ᚔᚈᚔ ᚍᚂᚐᚅᚑ ᚅᚔᚋᚌᚓᚅᚐ᚜" -- Old Irish (Ogham)
```
Comment (amosr) [inline disorder-corpus/src/Disorder/Corpus.hs]: some of them show up as boxy question marks for me. is that just a chrome thing? do they show up ok for you in text editor?

**Comment by amosr** [comment]
:doubleplusgood:

File: disorder-corpus/src/Disorder/Corpus.hs (hunk 1 above)
Comment (HuwCampbell) [inline disorder-corpus/src/Disorder/Corpus.hs]: These are ok in Firefox.

I get boxes for the Gothic ones though.

Comment (tmcgilchrist) [inline disorder-corpus/src/Disorder/Corpus.hs]: Safari is fine 👍

Comment (jystic) [inline disorder-corpus/src/Disorder/Corpus.hs]: yeah it's a problem with the font I think, here's what it's supposed to look like:

screen shot 2017-01-18 at 10 09 45 am

Comment (thumphries) [inline disorder-corpus/src/Disorder/Corpus.hs]: They're designed as a browser torture test. If you're seeing boxes, you're hurting your browser and should stop

**Comment by thumphries** [comment]
:⠓:

**Comment by olorin** [comment]
:100:

**Pull request merged**